portal/src/main/webapp/js/src/pancancer_study_summary/pancancer_study_summary.js

```diff
@@ -828,9 +828,11 @@ function DataManagerPresenter(dmInitCallBack)
 				{
 					//track cancer type detailed per cancer type:
 					var cancerType = sampleIdAndCancerTypeIdx[sampleClinicalData[i].sample];
-					if (!cancerType.cancerTypeDetailed[sampleClinicalData[i].attr_val])
-						cancerType.cancerTypeDetailed[sampleClinicalData[i].attr_val] = {sampleIds: []};
-					cancerType.cancerTypeDetailed[sampleClinicalData[i].attr_val].sampleIds.push(sampleClinicalData[i].sample);
+                    if (cancerTypeDetailed in cancerType) {
```

Comment:
@jjgao is this because there are samples with only CANCER_TYPE_DETAILED and without CANCER_TYPE? Is this expected? Or should we look for the root cause elsewhere?